go/enclave/evm/evm_facade.go

```diff
@@ -95,8 +95,9 @@ func ExecuteOffChainCall(msg *types.Message, s *state.StateDB, header *common.He
 
 	result, err := gethcore.ApplyMessage(vmenv, msg, gp)
 	if err != nil {
-		// this error is ignored by geth. logging just in case
+		// also return the result as the result can be evaluated on some errors like ErrIntrinsicGas
 		logger.Error("ErrKey applying msg:", log.ErrKey, err)
+		return result, fmt.Errorf("unable to ApplyMessage - %w", err)
```

Comment:
this has to be checked with @moraygrieve .
My feeling is this will break the other errors